compiler/util/clangUtil.cpp

```diff
@@ -856,6 +858,14 @@ CREATE_AST_CONSUMER_RETURN_TYPE CCodeGenAction::CreateASTConsumer(
 #endif
 };
 
+static void finishClang(GenInfo* info){
+  if( info->cgBuilder ) {
+    info->cgBuilder->Release();
+  }
+  info->Diags.reset();
+  info->DiagID.reset();
+}
```

Comment:
This looks like old code / merge error since finishClang contains a subset of cleanupClang.